sos/report/plugins/mysql.py

```diff
@@ -98,14 +99,25 @@ def setup(self):
 class DebianMysql(Mysql, DebianPlugin, UbuntuPlugin):
 
     packages = (
-        'mysql-server',
+        'mysql-server.*',
```

Comment:
What are the other package names expected here for mysql-server?

Reply:
'mysql-server' is a virtual package which depends on the current/latest server package version, with 2 package names that could be installed independently, the virtual package may not always be present:
mysql-server-8.0
mysql-server-core-8.0
(there are also versions for -5.6, -5.7, etc)